doc/pages/highlighters.asciidoc

```diff
@@ -93,27 +74,85 @@ add-highlighter window regex //\h*(TODO:)[^\n]* 0:cyan 1:yellow,red
     *-width <max_width>*:::
         wrap text at *max_width* if the window is wider.
 
+== General highlighters
+
 *fill* <face>::
     fill using the given *face*, mostly useful with regions highlighters
 
-*ranges* <option_name>::
-    use the data in the range-specs option of the given name to highlight
-    the buffer. The string part of the is interpretted as a face to apply
-    to the range.
-
-*replace-ranges* <option_name>::
-    use the data in the range-specs option of the given name to highlight
-    the buffer. The string part of the is interpretted as a display line to
-    display in place of the range.
-
 *column* <number> <face>::
     highlight column *number* with face *face*
 
 *line* <number> <face>::
     highlight line *number* with face *face*
 
-== Highlighting Groups
+*regex* <regex> <capture_id>:<face> ...::
+    highlight a regex, takes the regex as first parameter, followed by
+    any number of face parameters. For example:
+
+-------------------------------------------------------------------
+add-highlighter window regex //\h*(TODO:)[^\n]* 0:cyan 1:yellow,red
+-------------------------------------------------------------------
+
+    will highlight C++ style comments in cyan, with an eventual 'TODO:'
```

Comment:
In the GitHub preview, this paragraph is rendered in fixed-width like the `add-highlighter` command. I'm not sure if that's a problem with the markup or with GitHub's asciidoc support.